### PR TITLE
[Mime] Add support for application/vnd.etsi.asic-e+zip mime type

### PR DESCRIPTION
**Pull request by mllnd**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        |  -

`application/vnd.etsi.asic-e+zip` is a mime type used by `.asice` (formerly `.bdoc`) file containers. Such files are widely used in Estonia to represent digitally signed files.

**Resources:**
https://www.id.ee/wp-content/uploads/2020/06/bdoc-spec212-eng.pdf (spec)
https://www.iana.org/assignments/media-types/application/vnd.etsi.asic-e+zip

### CODE REVIEW

**Comment by stof** [comment]
This list is autogenerated from 2 sources:
- the upstream list of the Apache project
- customizations maintained in the generation script

The current contribution would be lost when re-running the script as it edits the generated list directly.

File: src/Symfony/Component/Mime/MimeTypes.php [hunk 1]
```diff
@@ -1849,7 +1851,7 @@ public function guessMimeType(string $path): ?string
         'bdf' => ['application/x-font-bdf'],
         'bdm' => ['application/vnd.syncml.dm+wbxml', 'video/mp2t'],
         'bdmv' => ['video/mp2t'],
-        'bdoc' => ['application/bdoc', 'application/x-bdoc'],
+        'bdoc' => ['application/bdoc', 'application/x-bdoc', 'application/vnd.etsi.asic-e+zip'],
```
Comment (stof) [inline src/Symfony/Component/Mime/MimeTypes.php]: This change looks wrong to me. the IANA assignment lists `asice` and `sce` as extensions, not `bdoc`

Comment (mllnd) [reply src/Symfony/Component/Mime/MimeTypes.php]: > This list is autogenerated from 2 sources:
> 
> * the upstream list of the Apache project
> * customizations maintained in the generation script
> 
> The current contribution would be lost when re-running the script as it edits the generated list directly.

By looking at the update script, it seems to me that old values are never actually removed...

https://github.com/symfony/symfony/blob/694c052e3c500ea81f7b602b94b53d322d100145/src/Symfony/Component/Mime/Resources/bin/update_mime_types.php#L63-L64

Comment (mllnd) [reply src/Symfony/Component/Mime/MimeTypes.php]: > This change looks wrong to me. the IANA assignment lists `asice` and `sce` as extensions, not `bdoc`

The file container's former extension was `.bdoc` (with the same mime type) with DigiDoc v3, hence I've added it there as well. The new version with DigiDoc v4 is `.asice`.

Here is a bit more context regarding both of these extensions: https://www.id.ee/en/article/what-is-the-difference-between-digitally-signed-documents-with-bdoc-and-asice-extensions/

**Review by nicolas-grekas — CHANGES_REQUESTED** [review]
I updated the types maps in https://github.com/symfony/symfony/pull/41016 and `application/vnd.etsi.asic-e+zip` is not part of it.

Please submit a change to the `src/Symfony/Component/Mime/Resources/bin/update_mime_types.php` file instead.

**Comment by mllnd** [comment]
@nicolas-grekas I've updated the PR and moved the additional extension along with its mime type to the update script.

**Review by nicolas-grekas — APPROVED** [review]
(I synced the DB)

**Comment by nicolas-grekas** [comment]
Thank you @mllnd.

**Pull request merged**